lib/alex_koin/slack_commands.ex

```diff
@@ -80,8 +80,39 @@ defmodule AlexKoin.SlackCommands do
     # 1. get coins to transfer
     coins = Repo.all(Coin.for_wallet(from_wallet, amount))
 
-    # 2. move the coins over to the other wallet
-    Enum.each(coins, fn(c) -> transfer_coin(c, from_wallet, to_wallet, memo) end)
+    # 2. Create the transaction, for now use the first coin's id
+    #    The coin id is legacy when we could only transfer 1 koin at a time
+    txn = %{
+        amount: amount,
+        memo: memo,
+        from_id: from_wallet.id,
+        to_id: to_wallet.id,
+        coin_id: List.first(coins).id
+      }
+    |> Transaction.changeset()
+    |> Repo.insert!()
+
+    # 3. Move the koins
+    # Pull the to_wallet again in case what we have is stale
```

Comment:
[non-blocker] Does the wallet get stale if it does some activity in behind our back (i.e, create a koin or transfer to someone else)? 
Does elixir have a concept of holding a lock over a database record? If yes, would it be better to use that instead since we also update wallet balance on L112 and it can get stale there as well?